src/index.js

```diff
@@ -576,13 +576,21 @@ import {
                     responseText = getRandomNegative();
                 }
                 else if (['help', 'command', 'info'].some(x => content.includes(x))) {
-                    responseText = '\n' + ['Examples of election FAQs I can help with:',
-                        'what is an election', 'how to nominate myself',
-                        'how to vote', 'who should I vote for',
-                        'how is candidate score calculated', 'what is my candidate score',
+                    responseText = '\n' + [
```

Comment:
I also have an idea of auto-generating help akin the help generated by the `CommandManager`, but it obviously requires finishing the reducer first

Reply:
As discussed, we can have a separate command to show all, e.g.:
`@electionbot expanded help`